src/main/java/com/gildedrose/DefaultItem.java

```diff
@@ -1,28 +1,30 @@
 package com.gildedrose;
 
-public class DefaultItem {
+public class DefaultItem implements Ageable {
 
+    private final Item item;
     protected int qualityDegradation;
 
-    public DefaultItem() {
+    public DefaultItem(Item item) {
+        this.item = item;
         this.qualityDegradation = 1;
     }
 
-    public void age(Item item) {
-        item.quality = updateQuality(item.quality, item.sellIn);
-        item.sellIn = updateSellIn(item.sellIn);
+    public void age() {
+        item.quality = updateQuality();
+        item.sellIn = updateSellIn();
     }
 
-    private int updateQuality(int quality, int sellIn) {
-        if (sellIn <= 0) {
+    private int updateQuality() {
+        if (item.sellIn <= 0) {
             this.qualityDegradation = 2;
         }
-        quality -= this.qualityDegradation;
-        return Math.max(quality, 0);
+        item.quality -= this.qualityDegradation;
```

Comment:
**Suggestion**: Possible to reduce the amount of if... else... statements w/ ternaries or something else more concise?

Reply:
Same question as above in the earlier comment on this subject